docs/source/logging/logging.md

```diff
@@ -1,54 +1,121 @@
+# Default logging configuration
+Kedro's [default logging configuration](https://github.com/kedro-org/kedro/blob/main/kedro/framework/project/default_logging.yml) defines a handler called `rich` that uses the Rich logging handler to format messages. We also use the Rich traceback handler to render exceptions.
 
-# Default framework-side logging configuration
+By default, Python only shows logging messages at level `WARNING` and above. Kedro's logging configuration specifies that `INFO` level messages from Kedro should also be emitted. This makes it easier to track the progress of your pipeline when you perform a kedro run.
 
-Kedro's [default logging configuration](https://github.com/kedro-org/kedro/blob/main/kedro/framework/project/default_logging.yml) defines a handler called `rich` that uses the [Rich logging handler](https://rich.readthedocs.io/en/stable/logging.html) to format messages. We also use the [Rich traceback handler](https://rich.readthedocs.io/en/stable/traceback.html) to render exceptions.
+## How to perform logging in your Kedro project
+To add logging to your own code (e.g. in a node):
 
-By default, Python only shows logging messages at level `WARNING` and above. Kedro's logging configuration specifies that `INFO` level messages from Kedro should also be emitted. This makes it easier to track the progress of your pipeline when you perform a `kedro run`.
+```python
+import logging
 
-## Project-side logging configuration
+log = logging.getLogger(__name__)
+log.warning("Issue warning")
+log.info("Send information")
+log.debug("Useful information for debugging")
+```
 
-In addition to the `rich` handler defined in Kedro's framework, the [project-side `conf/base/logging.yml`](https://github.com/kedro-org/kedro/blob/main/kedro/templates/project/%7B%7B%20cookiecutter.repo_name%20%7D%7D/conf/base/logging.yml) defines two further logging handlers:
-* `console`: show logs on standard output (typically your terminal screen) without any rich formatting
-* `info_file_handler`: write logs of level `INFO` and above to `info.log`
+```{note}
```

Comment:
Looking at this again, I realised I have no idea whatsoever what this means:

"The name of a logger corresponds to a key in the loggers section of the logging configuration file (e.g. kedro). See [Python’s logging documentation](https://docs.python.org/3/library/logging.html#logger-objects) for more information."

Does this have an relevance to the reader at this point or shall we remove it? Should it actually be a comment in the Kedro default_logging.yml file?

Reply:
I think this is explaining the line `log = logging.getLogger(__name__)`, here is how you define the `name` of a logger, and if you need configure it later what level of log you would like to see, you will do something like that. This is not kedro specific.
```yml
loggers:
  kedro:
    level: INFO
  name_of_my_logger:
    level: ERROR
```

If someone use a random name like `log = logging.getLogger("random_name")`, the above configuration will not have any effect. I think there is some value to educate user or provide a pointer to user, but we shouldn't try to explain too much about how Python logging work.